src/htm/types/Types.hpp

```diff
@@ -156,6 +157,10 @@ static const htm::Real32 Epsilon = htm::Real(1e-6);
 #endif
 
 
+  static const UInt SizeInt = sizeof(Int);
+  static const UInt shftInt = (SizeInt*CHAR_BIT)-1;
```

Comment:
ideally won't be needed. if it is, define it only in the file where needed.

Reply:
> ideally won't be needed. if it is, define it only in the file where needed.

Ok.